metomi/rose/host_select_client.py

```diff
@@ -22,7 +22,40 @@
 import psutil
 
 
+def usage():
+    print('''
+rose host-select-client
+
+Internal command for obtaining information about hosts using the
+Python psutil module.
+
+The names of psutil methods are passed to stdin along with any arguments
+as a JSON list.
+
+EXAMPLES:
+    # return the virtual memory and cpu usage
+    [["virual_memory"], ["cpu_percent"]]
+
+    # return disk usage for the filesystem mounted at "/"
+    [["disk_usage", "/"]]
+
+The input json should be preceded by \n**start**\n
+and followed by \n**end**\n to avoid issues with user profile scripts
+and stdin deadlock.
+
+$ rose host-select-client <<'__HERE__'
```

Comment:
I dont think that this worked when I tried it...

Reply:
Needed to remove the `$PS2` char when copying into the CLI.